### DISCUSSION TITLE
rebaseStalePrs setting not being respected

### DISCUSSION THREAD

**Discussion by bcoe** [renovatebot/renovate#17773] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

To reduce load on GitHub Actions, we set the renovate setting:

`"rebaseStalePrs": false`

With the hope that this would prevent outstanding renovate PRs from being rebased (kicking off our test suite). After setting this setting, we continue to see renovate rebase PRs (See: force push in [this PR](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/pull/2750))

Here is our current configuration, for the repository [GoogleCloudPlatform/nodejs-docs-samples](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/renovate.json):

```
{
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ],
  "pinVersions": false,
  "rebaseStalePrs": false
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment]
that option is deprecated, please configure `rebaseWhen`[^1]

[^1]: https://docs.renovatebot.com/configuration-options/#rebasewhen

**Comment by HonkingGoose** [comment, marked as answer]
@bcoe If you don't want Renovate to rebase your PRs then use this `renovate.json` config:

```json5
{
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ],
  "pinVersions": false,
  "rebaseWhen": "never" # <--- this is the change you want
}
```

If you want Renovate to rebase, you'll have to request it by clicking this checkbox in the PR:

> - [ ] If you want to rebase/retry this PR, click this checkbox.

**Comment by bcoe** [comment]
Thank you @HonkingGoose 👍
